src/momento/aio/_scs_control_client.py

```diff
@@ -70,5 +80,58 @@ async def list_caches(self, next_token: Optional[str] = None) -> ListCachesRespo
         except Exception as e:
             raise _cache_service_errors_converter.convert(e)
 
+    async def create_signing_key(
+        self, ttl_minutes: int, endpoint: str
+    ) -> CreateSigningKeyResponse:
+        _validate_ttl_minutes(ttl_minutes)
+        try:
```

Comment:
nit: You could make this code read a little nicer with a decorator so you're not copypasting try:except->converter.convert all over the place.